bindings/rust/wasmedge-sys/Cargo.toml

```diff
@@ -21,11 +21,15 @@ rand = "0.8.4"
 thiserror = "1.0.30"
 wasmedge-macro = {path = "../wasmedge-macro", version = "0.1"}
 wasmedge-types = {path = "../wasmedge-types", version = "0.3"}
+wasmtime-fiber = "1"
```

Comment:
What about using `wasmtime-fiber = "2"`? Seems the latest version of `wasmtime-fiber` is `2.0+`. Thanks!